scripts/eosio_build_ubuntu.sh

```diff
@@ -1,430 +1,259 @@
-	OS_VER=$( grep VERSION_ID /etc/os-release | cut -d'=' -f2 | sed 's/[^0-9\.]//gI' )
-	OS_MAJ=$(echo "${OS_VER}" | cut -d'.' -f1)
-	OS_MIN=$(echo "${OS_VER}" | cut -d'.' -f2)
+if [ $1 == 1 ]; then answer=1; fi # NONINTERACTIVE
```

Comment:
How is this designed to work? Won't arbitrary options to the build script cause noninteractive mode to be enabled?

Reply:
eosio_build.sh has `. "$FILE" $NONINTERACTIVE # Execute OS specific build file` and then above that earlier in the script it will default to NONINTTERACTIVE=0 if arg0 is not "noninteractive". We can always change this (using --noninteractive,etc), but it's useful for automated pipeline/image creations purposes.

Reply:
okay I see what you did there. We should probably integrate the option into the rest of the getopts, and no need to pass it as a parameter since the eosio_build_* scripts are sourced. Can hit this after merge.

Reply:
Sounds good